src/report_workflow/test_report_runner.py

```diff
@@ -135,4 +137,19 @@ def generate_test_command(test_type: str, test_manifest_path: str, artifacts_pat
     return command
 
 
+def get_os_cluster_logs(base_path: str, test_number: str, test_type: str, component_name: str, config: str) -> List[str]:
+    os_stdout: str = ''
+    os_stderr: str = ''
+    cluster_id: str = 'id-0' if config == 'with-security' else 'id-1'
```

Comment:
This hard code id will not work for the dashboards cases since dashboards will include two local cluster logs; it uses 0&1 for with-security and 2&3 for without-security.

Reply:
Fixed this. The stdout and stderr fields in the test report yaml will now be a list showing appropriate cluster-ids for OS and OSD.